opentech/apply/dashboard/tests/test_views.py

```diff
@@ -61,3 +61,29 @@ def test_cannot_see_submission_in_determination_when_not_lead(self):
         ApplicationSubmissionFactory(status='concept_review_discussion', workflow_stages=2, form_data__title='Reviewr')
         response = self.get_page()
         self.assertNotContains(response, 'Reviewr')
+
+    def test_waiting_for_review_with_count(self):
+        submission = ApplicationSubmissionFactory(status='external_review', workflow_stages=2, reviewers=[self.user])
```

Comment:
Hmm, i think this is an error. Staff should not be reviewing during the external review phase. This is a preexisting issue with the view.

Reply:
I can remove the test if it's a pre-existing issue... or try to fix if you tell me more about it. Let me know how to move forward.

Reply:
Lets leave it for now.